WaysofWork.md

```diff
@@ -1,169 +1,184 @@
 
-The issue will be permanent and updated via comments to avoid drowing the repo, meeting minutes will not be reflected here but stored in Sharepoint and as files in this repo. This issue will therefore only track status of issues so PM can maintian the team's Project.
+During monthly review meeting the team will go through the stories and share updates based on what reporting is written on them.
 
-From Thursdays and before the weekly meeting (Monday 10 AM) each team member will have to:
-- Make updates in the issues they have worked on via comments
-- Change their status in the project where appropiate
-- Update fields where appropiate
-- Fill in the meeting collaborative note here: https://hackmd.io/nPewceMuQcmXD7jfqYFpzw?edit
+#### Tasks and other issues from your day to day
+Stories are intended to link and group to our day to day work, allowing reporting at a more aggregated level.
+It is not required to create issues for every task but do so every time it is appropiatte.
 
-The weekly standup will then take place, discussing blockers and planning work.
-Team members should update the collaborative note as appropiate to reflect plans for the week and any agreed actions
+Link issues, milestones or other documents and links to stories to which they contribute to.
 
-The RPM will:
-- Update the weekly issue to reflect the team overall work for the last week and plans for this week
-- Review roadmap and boards to ensure it reflects the current project status
-- Add any necessary points for discussion for the monthly meeting
+Try to ask the following question each time you start a task: does it contribute to an ongoing or planned story?
 
-A member of the REG team will be asked to archive the meeting minute in the collaborative note in Sharepoint [here](https://thealanturininstitute.sharepoint.com/sites/SafeHaven/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FSafeHaven%2FShared%20Documents%2FMeetings&viewid=10151919%2Deeef%2D4a8b%2Db4c0%2D26a3b155773b)
+##### I do not know what Story this issue belongs to, Should I create this issue?
+All work and the related issues should clearly contribute to agreed upon goals, and their corresponding stories.
+If you are unable to assign an issue to a story consider whether to create it in the first place.
+If you think it is work that the project needs, add it for discussion during the next weekly meeting or open a thread in the corresponding channel.
 
-##### Monthly issue
 
-The monthly issue will be used to update the [month by month report](https://github.com/alan-turing-institute/data-safe-haven-team/blob/main/Reports/Monthly.md),
-as well as put together the agenda for the monthly meeting.
+If this happens regularly, it should be addressed.
+Either there is a deviation from plans and goals or the plans and goals are losing validity.
+The next monthly meeting should be longer, at least two hours, and focused on the roadmap and project strategy rather than progress.
```

Comment:
Will see if we get to this below, is there something here about quarterly review of project strategy too?

Reply:
Yeah, we will see how we actually go about this. If we actually extend monthlies, or have an ad hoc meeting, or if one hour is enough.

There is nothing (yet) on periodic strategy sessions or review, do you think we should?

Feel free to add a line or comment on how this is an ongoing process, or waht we are considering